docs/proposals/20200220-cluster-resource-set.md

```diff
@@ -121,13 +129,16 @@ spec:
      kind: ConfigMap
 ```
 
-Initially, the only supported mode will be `ApplyOnce` and it will be the default mode if no mode is provided. In the future, we may consider adding a `Reconcile` mode that reapplies the resources on resource hash change and/or periodically.
+The supported modes will be: 
+* `ApplyOnce`. This will be the default mode if no mode is provided. Resources are only applied one.
+* `Reconcile`. Resources are reapplied when the content of the `ConfigMap` or `Secret` that defines them changes.
```

Comment:
Maybe we can add a note to explicitly state addition of `Reconcile` mode doesn't mean we will handle deleting resources.